doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md

```diff
@@ -4,5 +4,79 @@ title: Intel Edison
 permalink: /:path/
 sort_idx: 60
 ---
+{% include variables.md %]
 
-SDK integration instructions for select platforms
+This guide explains how to build applications for Intel Edison based on the Kaa C++ endpoint SDK (further, the C++ SDK).
+
+**Note:** The instructions below are for the official Intel Edison firmware.
+
+## Configuring Intel Edison board
+
+If this is the first time you use the Intel Edison technology, you have to start with configuring your board. For this purpose, refer to the official Edison getting started guide.
+After finishing with the guide, you will be able to access your board through both a serial terminal and an ssh client.
+
+## Installing third-party components for C++ SDK
+
+The following third-party components must be installed on the Intel Edison board before building the C++ SDK.
+
+* **Mandatory:** [Boost](http://www.boost.org/) (1.54 min.v.), [Avro](http://avro.apache.org/) (1.7.5 - 1.7.7 v.) and [Botan](https://botan.randombit.net/)  (only v.1.10) libraries.
+* **Optional:** [SQLite](https://www.sqlite.org/) library. It should be installed only if you are going to use a persistent log storage for [the Kaa data collection feature](https://docs.kaaproject.org/display/KAA/Collecting+data+from+endpoints).
```

Comment:
`https://docs.kaaproject.org/display/KAA/Collecting+data+from+endpoints` is old link. Use new one.